api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -252,7 +252,7 @@ void shouldSendNewChildEmailWhenPaymentCycleIncludesBackdatedVouchers() throws J
         wiremockManager.stubSuccessfulDepositResponse(CARD_ACCOUNT_ID);
         stubNotificationEmailResponse();
 
-        Claim claim = createActiveClaimWithPaymentCycleEndingYesterday(SINGLE_THREE_YEAR_OLD, LocalDate.now().minusWeeks(7));
+        Claim claim = createActiveClaimWithPaymentCycleEndingYesterday(SINGLE_THREE_YEAR_OLD, LocalDate.now().minusWeeks(9));
```

Comment:
Changed from 7 to 9 weeks so that a report a birth email is not sent in this test. Sending a report a birth email is covered in another test.